src/contexts/NetworkContext.tsx

```diff
@@ -125,31 +168,61 @@ const NetworkContextProvider = ({ children }: NetworkContextProps) => {
       setIsLight(true)
       const smoldot = startFromWorker(new SmWorker())
       let relayChain: Promise<Chain>
+      let peopleChain: Promise<Chain>
       if (relay === 'polkadot') {
         relayChain = import('polkadot-api/chains/polkadot').then(
           ({ chainSpec }) => smoldot.addChain({ chainSpec }),
         )
+
+        peopleChain = Promise.all([
+          relayChain,
+          import('polkadot-api/chains/polkadot_people'),
+        ]).then(([relayChain, { chainSpec }]) =>
+          smoldot.addChain({ chainSpec, potentialRelayChains: [relayChain] }),
+        )
       } else {
         relayChain = import('polkadot-api/chains/ksmcc3').then(
           ({ chainSpec }) => smoldot.addChain({ chainSpec }),
         )
+
+        peopleChain = Promise.all([
+          relayChain,
+          import('polkadot-api/chains/ksmcc3_people'),
+        ]).then(([relayChain, { chainSpec }]) =>
+          smoldot.addChain({ chainSpec, potentialRelayChains: [relayChain] }),
+        )
       }
 
       client = createClient(getSmProvider(relayChain))
+      peopleClient = createClient(getSmProvider(peopleChain))
     } else {
       const { assetInfo, wsEndpoint } = getChainInformation(network)
       setAssetInfo(assetInfo)
       setIsLight(false)
 
       client = createClient(getWsProvider(wsEndpoint))
+      let wss: string = ''
+      if (network === 'polkadot') {
+        wss = 'wss://sys.ibp.network/people-polkadot'
+      } else if (network === 'kusama') {
+        wss = 'wss://sys.ibp.network/people-kusama'
+      } else {
+        wss = 'wss://sys.ibp.network/people-westend'
+      }
+      peopleClient = createClient(getWsProvider(wss))
     }
 
     const descriptor = descriptorName[network]
     const typedApi = client.getTypedApi(descriptor)
 
+    const descriptorPeople = descriptorPeopleName[peopleNetwork!]
```

Comment:
No need of an assertion here, it's set in any case.
```suggestion
    const descriptorPeople = descriptorPeopleName[peopleNetwork]
```